src/main.js

```diff
@@ -23,6 +24,7 @@ export default function Main() {
 			<Route exact path="/contact" component={Contact}/>
 			<Route exact path="/discovery" component={Discovery}/>
 			<Route exact path="/glossary" component={Glossary}/>
+			{/* <Route exact path="/donate" component={Donate}/> */}
```

Comment:
This and the above import should be removed from this PR as they are not ready for release.